frontend_tests/node_tests/narrow_local.js

```diff
@@ -325,7 +325,7 @@ run_test('search', () => {
         },
         expected_id_info: {
             target_id: undefined,
-            final_select_id: undefined,
+            final_select_id: 10000000000000000,
```

Comment:
Just to be clear, after talking with @thedeveloperr, this is the change in the tests that should keep this issue from showing up in the future without being detected.